spec/controllers/users/totp_setup_controller_spec.rb

```diff
@@ -32,10 +30,6 @@
           subject.current_user.decorate.qrcode(subject.user_session[:new_totp_secret])
         ).not_to be_nil
       end
-
-      it 'presents a QR code to the user' do
```

Comment:
I thought we had a feature spec for this, but I don't see one. It's probably a good idea to have one for this.